data/mods/MindOverMatter/powers/electrokinesis.json

```diff
@@ -531,6 +531,49 @@
     "targeted_monster_species": [ "CYBORG", "ROBOT" ],
     "ignored_monster_species": [ "YRAX_CONSTRUCT", "PSI_NULL" ]
   },
+  {
+    "id": "electrokinetic_robot_interface",
+    "type": "SPELL",
+    "name": "[Ψ]Robotic Interface",
+    "description": "Use your powers to remotely interface with a robot's internal circuitry, allowing you to hack it if you have the skill.",
+    "teachable": false,
+    "valid_targets": [ "self" ],
+    "spell_class": "ELECTROKINETIC",
+    "skill": "metaphysics",
+    "flags": [ "PSIONIC", "CONCENTRATE", "SILENT", "NO_HANDS", "NO_LEGS", "NO_EXPLOSION_SFX" ],
+    "difficulty": 10,
+    "max_level": { "math": [ "int_to_level(1)" ] },
+    "effect": "effect_on_condition",
+    "effect_str": "EOC_ELECTROKIN_ROBOT_INTERFACE",
+    "extra_effects": [ { "id": "psionic_drained_difficulty_eight", "hit_self": true } ],
+    "shape": "blast",
+    "min_damage": 1,
```

Comment:
```suggestion

```
This (and max_damage) shouldn't be necessary because the actual spawning is handled by EoC (and if left in, they'll end up with an electrokinetic using it and shattering their ballistic plates, which is definitely a mistake I've made in the past...)